src/content/developers/tutorials/hello-world-smart-contract/index.md

```diff
@@ -0,0 +1,350 @@
+---
+title: Hello World Smart Contract for Beginners
+description: Introductory tutorial on writing and deploying a simple smart contract on Ethereum.
+author: "elanh"
+tags: ["solidity", "hardhat", "alchemy", "smart contracts", "getting started", "deploying"]
+skill: beginner
+lang: en
+sidebar: true
+published: 2021-03-31
+---
+
+If you are new to blockchain development and don’t know where to start, or if you just want to understand how to deploy and interact with smart contracts, this guide is for you. We will walk through creating and deploying a simple smart contract on the Ropsten test network using a virtual wallet ([Metamask](https://metamask.io/)), [Solidity](https://docs.soliditylang.org/en/v0.8.0/), [Hardhat](https://hardhat.org/), and [Alchemy](https://alchemyapi.io/eth) (don’t worry if you don’t understand what any of this means yet, we will explain it).
+
+In part 2 of this tutorial we’ll go through how we can interact with our smart contract once it’s deployed, and in part 3 we’ll cover how to publish it on Etherscan.
+
+If you have questions at any point feel free to reach out in the [Alchemy Discord](https://discord.gg/gWuC7zB)!
+
+## Step 1: Connect to the Ethereum network {#step-1}
+
+There are many ways to make requests to the Ethereum chain. For simplicity, we’ll use a free account on Alchemy, a blockchain developer platform and API that allows us to communicate with the Ethereum chain without having to run our own nodes. The platform also has developer tools for monitoring and analytics that we’ll take advantage of in this tutorial to understand what’s going on under the hood in our smart contract deployment. If you don’t already have an Alchemy account, [you can sign up for free here](https://dashboard.alchemyapi.io/signup).
+
+## Step 2: Create your app (and API key) {#step-2}
+
+Once you’ve created an Alchemy account, you can generate an API key by creating an app. This will allow us to make requests to the Ropsten test network. If you’re not familiar with testnets, check out [this page](/developers/docs/networks/).
+
+1.  Navigate to the “Create App” page in your Alchemy Dashboard by hovering over “Apps” in the nav bar and clicking “Create App”
+
+![Hello world create app](./hello-world-create-app.png)
+
+1. Name your app “Hello World”, offer a short description, select “Staging” for the Environment (used for your app bookkeeping), and choose “Ropsten” for your network.
```

Comment:
🤦🏻 My apologies... using the `1.` `1.` `1.`  numbering only works if nothing is between them. Since you have the images here, I would just change this back to `1.` `2.` `3.` 
Sorry about that